src/store-tree/index.ls

```diff
@@ -71,6 +71,12 @@ class StoreTree extends StoreNode
     obj
 
 
+  $debug: ->
+    obj = {}
+    @_for-each-subnode (subnode, key) -> obj[key] = subnode.$debug!
```

Comment:
Cool. I'll add `prelude-ls` in a follow up PR and try to clean up a bunch of stuff